cli/context/kubernetes/save.go

```diff
@@ -39,6 +39,13 @@ func FromKubeConfig(kubeconfig, kubeContext, namespaceOverride string) (Endpoint
 			Key:  key,
 		}
 	}
+	var usernamePassword *UsernamePassword
+	if clientcfg.Username != "" || clientcfg.Password != "" {
```

Comment:
~Do we need the check here? Of both are empty, we'll just set an empty string (which is the default for strings)~

oh, nevermind; you created a pointer so that we don't set the `UsernamePassword` field if we don't use it